src/docs/tests/validators/example-validator.ts

```
@@ -0,0 +1,48 @@
+    }
+
+    const generatedSchemas = openApiSpec.components?.schemas || {};
+
+    const schemaClasses = {
+      ProductSchema,
+      OrderSchema,
+      OrderItemSchema,
+      UserSchema,
+    };
+    //IF npm install ts-json-schema-generator ajv
+    for (const [schemaName, SchemaClass] of Object.entries(schemaClasses)) {
```

ESLint

[error] 'SchemaClass' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)